src/Lib.hs

```diff
@@ -231,6 +216,22 @@ extractEmailAddress ticketId = do
     liftIO $ appendFile "emailAddress.txt" (emailAddress <> "\n")
     liftIO $ putTextLn emailAddress
 
+getTickets :: App [TicketInfo]
```

Comment:
Who is this for?

Reply:
ShowStatistics           -> void $ runApp (getTickets >>= showStatistics) cfg